### PR TITLE
Add Ubuntu16.04 support.

### PR DESCRIPTION
**Pull request by Priya91**
cc @joshfree @ellismg @mmitche

### CODE REVIEW

**Comment by ellismg** [comment]
LGTM

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -124,6 +124,17 @@ class Utilities {
                                 // For outerloop runs.
                                 'outer-latest-or-auto':'auto-ubuntu1510-20160131outer'
                                 ],
+                            'Ubuntu16.04' :
```
Comment (joshfree) [inline jobs/generation/Utilities.groovy]: what about removing the Ubuntu15.10 section?

Comment (Priya91) [inline jobs/generation/Utilities.groovy]: Will do that after the Ubuntu16.04 runs are successful and fully supported with packages.

Comment (joshfree) [inline jobs/generation/Utilities.groovy]: OK.  LGTM then

**Comment by Priya91** [comment]
@ellismg @mmitche Can you merge the PR, I don't have write access

**Pull request merged**